opm/material/fluidsystems/blackoilpvt/GasPvtMultiplexer.hpp

```diff
@@ -160,6 +160,18 @@ class GasPvtMultiplexer
         }
     }
 
+    bool mixingEnergy(){
+        switch (gasPvtApproach_) {
+        case GasPvtApproach::ThermalGas: {
+            return false;
+            break;
+        }
+        default: {
+            return false;
+        }
+        }
```

Comment:
For CO2STORE and H2STORE the mixingEnergy is already included. Do you plan to extract that part out and return true here or just keep it as is and return false.

Reply:
I will return true for the cases which should use a mixing model. i.e. only for thermal blackoil?